specs/core/0_beacon-chain.md

```diff
@@ -411,36 +392,27 @@ The types are defined topologically to aid in facilitating an executable version
 
 ```python
 {
-    # BLS public key
+    # Fields relevant to light clients
```

Comment:
Not sure I agree with the grouping here. pubkey and slashed are relevant to far more than light clients